src/igvfd/types/sample.py

```diff
@@ -63,15 +63,15 @@ def sex(self, request, donors=None):
         elif len(sexes) > 1:
             return 'mixed'
 
-    @calculated_property(
-        schema={
-            'title': 'Age',
-            'description': 'Age of organism at the time of collection of the sample.',
-            'type': 'string',
-            'pattern': '^((\\d+(\\.[1-9])?(\\-\\d+(\\.[1-9])?)?)|(unknown)|([1-8]?\\d)|(90 or above))$',
-            'notSubmittable': True,
-        }
-    )
+    @calculated_property(define=True,
```

Comment:
why here it is "define=True" but in others, it is not?

Reply:
`define=True` is necessary here in `age` to be able to pass it into another calculated property, i.e. `summary`